units/sdl2_mixer.pas

```diff
@@ -171,6 +165,14 @@   TMix_Chunk = record
   {* Open the mixer with a certain audio format *}
 function Mix_OpenAudio(frequency: cint; format: cuint16; channels: cint; chunksize: cint): cint cdecl; external MIX_LibName {$IFDEF DELPHI} {$IFDEF MACOS} name '_MIX_OpenAudio' {$ENDIF} {$ENDIF};
 
+  { * Open a specific audio device for playback. *}
+function Mix_OpenAudioDevice(frequency: cint; format: cuint16; channels: cint; chunksize: cint; device: PAnsiChar; allowed_changes: cint): cint; cdecl;
```

Comment:
Again, the large comment is missing? Is this intentional? Seems to me to cover a few very interesting and important notes.

Reply:
Yes, it's intentional. Currently all SDL_Mixer functions have small comment stubs. I will follow this up with a separate PR which adds the original doc-comments for all symbols.